stories/index.js

```diff
@@ -16,12 +19,16 @@ storiesOf('Button', module).addParameters({
     <Button text="Hello World!"/>
 ));
 
-storiesOf('Image', module).addParameters({
+storiesOf('NavBar', module).addParameters({
```

Comment:
Why are use override Image story instead of adding new NavBar and keeping Image?

Reply:
that was a mistake, i think i reverted the commit and pushed again, but i guess i willl try again, or this can be solved in merging to develop i guess,